FluentUI.Demo/src/main/res/drawable/ic_fluent_add_24_regular.xml

```diff
@@ -0,0 +1,3 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android" android:width="24dp" android:height="24dp" android:viewportWidth="24" android:viewportHeight="24">
```

Comment:
To be clear about the design here, we will expect the consumer to provide the icons, so in order to use the Fluent icons the consumer should provide those independently?

Reply:
(and I presume not using the Fluent-System-Icons library instead of adding these resources is because it would add too much to the package size? it looks like these icons are all taken from that library, so any updates to the icons at source will not be reflected without manual updates)

Reply:
> To be clear about the design here, we will expect the consumer to provide the icons, so in order to use the Fluent icons the consumer should provide those independently?

We just add these icon resources for demonstration, I think we can delete these and using other icons instead

>(and I presume not using the Fluent-System-Icons library instead of adding these resources is because it would add too much to the package size? it looks like these icons are all taken from that library, so any updates to the icons at source will not be reflected without manual updates)

I think it doesn't increase any package size since these icons are only added in the Demo module instead of the Library module.

Reply:
Yes, that is correct. For our case we should include the icon resources with our UI library, rather than have the host do this. We would intend to use icons from the Fluent-System-Icons library also for this case. Does this approach seem correct to you?

Reply:
Do you mean we should add `com.microsoft.design:fluent-system-icons:version` into our `FluentUI.Demo` instead of importing icon resource?